Enterprise hub cookbook (#98)
huggingface/cookbook: notebooks/en/enterprise_cookbook_overview.md

# Enterprise Hub Cookbook

The Enterprise Hub Cookbook is designed for power users and enterprises who want to go beyond the standard free features of the Hugging Face Hub and integrate machine learning deeper into their production workflows. 

> The cookbook guides you through a selection of recipes (Jupyter Notebooks) with copy-pastable code to help you get started with advanced features of the Hub. For a quick overview of the Enterprise Hub, you can also watch [this 15 minute video](https://www.youtube.com/watch?v=CPQGBn-yXJQ). 

Reviewer: Could be nicer to just embed the video on the page :)

```suggestion
The cookbook guides you through a selection of recipes (Jupyter Notebooks) with copy-pastable code to help you get started with the Hub's advanced features.

<Youtube id="CPQGBn-yXJQ"/>
```

> ## 1. Interactive Development in HF Spaces [coming soon]

Reviewer: I think it might be better not to do an ordered list since this might give users the impression that they should follow these recipes in a certain order.

```suggestion
## Interactive Development in HF Spaces [coming soon]
```

> With JupyterLab Spaces you can spin up your personal Jupyter Notebook like in Google Colab, only with a wider selection of reliable CPUs and GPUs (e.g. H100 or 4xA10G) which you can select and switch on the fly. With Spaces `dev mode` you can also use this cloud hardware from your local IDE (e.g. VS Code). Read this recipe to learn how to spin up a GPU and connect to it via your local IDE.
> 
> See also our [JupyterLab Spaces docs](https://huggingface.co/docs/hub/spaces-sdks-docker-jupyter) and our `dev mode` beta feature [docs](https://huggingface.co/dev-mode-explorers).

Reviewer: ```suggestion
With JupyterLab Spaces, you can spin up your personal Jupyter Notebook like in Google Colab, only with a wider selection of more reliable CPUs and GPUs (H100 or 4xA10G) that you can select and switch on the fly. In the Spaces Dev Mode, you can use this cloud hardware from your local IDE (e.g., VS Code). Read this recipe to learn how to spin up a GPU and connect to it via your local IDE.

For more details, read the [JupyterLab Spaces](https://huggingface.co/docs/hub/spaces-sdks-docker-jupyter) and the [Dev Mode](https://huggingface.co/dev-mode-explorers) guides.
```

## 2. Inference API (Serverless) [coming soon]

> With our serverless Inference API, you can test a range of open source models with simple API calls (e.g. generative LLMs, efficient embedding models, or image generators). The serverless Inference API is rate limited and mostly intended for initial testing or low-volume use. Read this recipe to learn how to query the serverless Inference API.
> 
> See also our [serverless API docs](https://huggingface.co/docs/api-inference/index).

Reviewer: ```suggestion
With our serverless Inference API, you can test a range of open source models with simple API calls (e.g. generative LLMs, efficient embedding models, or image generators). The serverless Inference API is rate limited and mostly intended for initial testing or low-volume use. Read this recipe to learn how to query the serverless Inference API.

For more details, read the [serverless API](https://huggingface.co/docs/api-inference/index) guide.
```

## 3. Inference Endpoints (dedicated) [coming soon]

> With our dedicated Inference Endpoints, you can easily deploy any model and on a wide range of hardware, essentially creating your personal production-ready API in a few clicks. Read this recipe to learn how to create and configure your own dedicated Endpoint.
> 
> See also our [Endpoint docs](https://huggingface.co/docs/inference-endpoints/index). 

Reviewer: ```suggestion
With our dedicated Inference Endpoints, you can easily deploy any model on a wide range of hardware, essentially creating your personal production-ready API in a few clicks. Read this recipe to learn how to create and configure your own dedicated Endpoint.

See the [Endpoint](https://huggingface.co/docs/inference-endpoints/index) guide to learn more. 
```

## 4. Data Annotation with Argilla Spaces [coming soon]

> Whether you're zero-shot testing an LLM or you're training your own model - creating good test or train data is maybe the highest value investment you can make at the beginning of your machine learning journey. Argilla is a free open-source data annotation tool that enables you to create high quality data for text, image or audio tasks. Read this recipe to learn how to create a data annotation workflow (alone or in a larger team) in your browser.
> 
> See also the [Argilla docs](https://docs.argilla.io/en/latest/) and the [docs](https://huggingface.co/docs/hub/spaces-sdks-docker-argilla) for the HF Argilla Spaces integration.

Reviewer: ```suggestion
Whether you're zero-shot testing an LLM or training your own model, creating good test or train data is maybe the highest-value investment you can make at the beginning of your machine learning journey. Argilla is a free, open-source data annotation tool that enables you to create high-quality data for text, image, or audio tasks. Read this recipe to learn how to create a data annotation workflow (alone or in a larger team) in your browser.

See the [Argilla](https://docs.argilla.io/en/latest/) documentation and the [HF Argilla Spaces](https://huggingface.co/docs/hub/spaces-sdks-docker-argilla) integration for more details.
```

## 5. AutoTrain Spaces [coming soon]

> With our AutoTrain Spaces you can train your own machine learning models in a simple interface without any code. Read this recipe to learn how to fine-tune your own LLM in an AutoTrain Space on the Hub on a wide range of GPUs. 
> 
> See also our [AutoTrain docs](https://huggingface.co/docs/autotrain/index).

Reviewer: ```suggestion
With AutoTrain Spaces, you can train your own machine learning models in a simple interface without any code. Read this recipe to learn how to fine-tune your own LLM in an AutoTrain Space on the Hub on a wide range of GPUs. 

See the [AutoTrain](https://huggingface.co/docs/autotrain/index) documentation to learn more.
```

> ## 6. Creating Private Demos with Spaces and Gradio [coming soon]

Reviewer: ```suggestion
## 6. Creating private demos with Spaces and Gradio [coming soon]
```

> Visual demos speak more than 1000 words. Demos are particularly important if you want to convince stakeholders of a machine learning MVP. Read this recipe to learn how to create a private machine learning demo on Spaces with Gradio.
> 
> See also our [Spaces docs](https://huggingface.co/docs/hub/spaces-overview) and the [Gradio Spaces docs](https://huggingface.co/docs/hub/spaces-sdks-gradio).

Reviewer: ```suggestion
Visual demos speak louder than words. Demos are particularly important if you want to convince stakeholders of a machine learning minimum viable product (MVP). Read this recipe to learn how to create a private machine learning demo on Spaces with Gradio.

See the [Spaces](https://huggingface.co/docs/hub/spaces-overview) and [Gradio Spaces](https://huggingface.co/docs/hub/spaces-sdks-gradio) documentation to learn more.
```

> ## 7. Advanced Collaboration on the Hub [coming soon]

Reviewer: ```suggestion
## 7. Advanced collaboration on the Hub [coming soon]
```

# Enterprise Hub Cookbook

The Enterprise Hub Cookbook is designed for power users and enterprises who want to go beyond the standard free features of the Hugging Face Hub and integrate machine learning deeper into their production workflows. The cookbook guides you through a selection of recipes (Jupyter Notebooks) with copy-pastable code to help you get started with the Hub's advanced features.

<Youtube id="CPQGBn-yXJQ"/>

> ## Interactive Development in HF Spaces \[coming soon\]

Reviewer: I think the brackets are typically used when there are two identical headers, but we want to differentiate the two. Let's try using parentheses instead?

Author: ok, updated it to the simpler "(coming soon)"
